priv/react_app/src/api/CompanyApi.js

```diff
@@ -0,0 +1,25 @@
+/*
+*   This file contains methods to access the /companies reasource on the server.
+*/
+
+
+class CompanyApi {
+
+	static apiUrl = process.env.REACT_APP_API_URL;
```

Comment:
Den här kommer balla ur om miljövariabeln inte är satt. Då kommer den skicka requests till `undefined/<rest-of-url>`.

Den känns onödig whatsoever. Om man gör `fetch(/api/companies)` så kommer den skicka till samma host som den hostas på: `https://arkad-nexpo.herokuapp.com` i produktion och `localhost:3000` i dev. Antar att du gjorde den här lösningen för att vi vill ha `localhost:4000` i dev.

Jag tycker det känns bättre att lösa problemet med en proxy i dev-miljön, istället för att införa en ny miljövariabel i produktion. Cool om jag pushar en commit för att visa vad jag menar?

Reply:
Chill, satte det mest för att kunna köra frontend helt separerat från backend via typ github pages eller liknande! Antar du menar att sätta proxy i package.json?